src/components/ADempiere/Form/VPOS/OrderList/index.vue

```diff
@@ -112,7 +112,11 @@
         width="120"
       >
         <template slot-scope="scope">
-          {{ formatQuantity(scope.row.grandTotal) }}
+          {{
```

Comment:
Just can see change for source code with the same functionality, please let it without changes